plugins/Organic/Organic.cpp

```diff
@@ -159,61 +155,58 @@ OrganicInstrument::~OrganicInstrument()
 
 
 
-void OrganicInstrument::saveSettings( QDomDocument & _doc, QDomElement & _this )
+void OrganicInstrument::saveSettings(QDomDocument& doc, QDomElement& elem)
 {
-	_this.setAttribute( "num_osc", QString::number( m_numOscillators ) );
-	m_fx1Model.saveSettings( _doc, _this, "foldback" );
-	m_volModel.saveSettings( _doc, _this, "vol" );
+	elem.setAttribute("num_osc", QString::number(m_numOscillators));
+	m_fx1Model.saveSettings(doc, elem, "foldback");
+	m_volModel.saveSettings(doc, elem, "vol");
 
-	for( int i = 0; i < m_numOscillators; ++i )
+	for (int i = 0; i < m_numOscillators; ++i)
 	{
-		QString is = QString::number( i );
-		m_osc[i]->m_volModel.saveSettings( _doc, _this, "vol" + is );
-		m_osc[i]->m_panModel.saveSettings( _doc, _this, "pan" + is );
-		m_osc[i]->m_harmModel.saveSettings( _doc, _this, "newharmonic" + is );
-
-		m_osc[i]->m_detuneModel.saveSettings( _doc, _this, "newdetune"
-									+ is );
-		m_osc[i]->m_oscModel.saveSettings( _doc, _this, "wavetype"
-									+ is );
+		QString is = QString::number(i);
+		m_osc[i]->m_volModel.saveSettings(doc, elem, "vol" + is);
+		m_osc[i]->m_panModel.saveSettings(doc, elem, "pan" + is);
+		m_osc[i]->m_harmModel.saveSettings(doc, elem, "newharmonic" + is);
+		m_osc[i]->m_detuneModel.saveSettings(doc, elem, "newdetune" + is);
+		m_osc[i]->m_oscModel.saveSettings(doc, elem, "wavetype" + is);
 	}
 }
 
 
 
 
-void OrganicInstrument::loadSettings( const QDomElement & _this )
+void OrganicInstrument::loadSettings(const QDomElement& elem)
 {
-//	m_numOscillators =  _this.attribute( "num_osc" ).
-	//							toInt();
-
-	for( int i = 0; i < m_numOscillators; ++i )
+	for (int i = 0; i < m_numOscillators; ++i)
 	{
-		QString is = QString::number( i );
-		m_osc[i]->m_volModel.loadSettings( _this, "vol" + is );
-		if( _this.hasAttribute( "detune" + is ) )
+		const auto is = QString::number(i);
+
+		m_osc[i]->m_volModel.loadSettings(elem, "vol" + is);
+
+		if (elem.hasAttribute("detune" + is))
```

Comment:
The "detune" key [hasn't been used since 2014](https://github.com/LMMS/lmms/commit/63747787e414b9511625dd6c61f028886bb213c7), but it should be fixed with the changes I just pushed.